config/robots.rb

```diff
@@ -3727,4 +3727,252 @@
         }
       }
     )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Verify initial setup
+    custom_robot(
+      'bravo-verify-initial-setup',
+      name: 'Bravo Verify Initial Setup',
+      require_robot: false,
+      beds: {
+        bed(4).barcode => {
+          purpose: 'LCMT Lysate',
+          states: ['passed'],
+          label: 'Bed 4'
+        },
+        car('1,4').barcode => {
+          purpose: 'LCMT DNA Frag',
+          states: ['pending'],
+          label: 'Carousel 1,4',
+          parent: bed(4).barcode
+        },
+        car('2,4').barcode => {
+          purpose: 'LCMT DNA End Prep',
+          states: ['pending'],
+          label: 'Carousel 2,4',
+          parent: car('1,4').barcode
+        },
+        car('3,5').barcode => {
+          purpose: 'LCMT DNA Adp Lig',
+          states: ['pending'],
+          label: 'Carousel 3,5',
+          parent: car('2,4').barcode
+        },
+        car('4,4').barcode => {
+          purpose: 'LCMT EM TET2 Ox',
+          states: ['pending'],
+          label: 'Carousel 4,4',
+          parent: car('3,5').barcode
+        },
+        car('4,3').barcode => {
+          purpose: 'LCMT EM TET2 Stop',
+          states: ['pending'],
+          label: 'Carousel 4,3',
+          parent: car('4,4').barcode
+        }
+      }
+    )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Bravo LCMT DNA Frag Verification
+    custom_robot(
+      'bravo-lcmt-dna-frag-verification',
+      name: 'Bravo LCMT DNA Frag Verification',
+      require_robot: false,
```

Comment:
needs verify_robot true

Reply:
All except initial setup and Hamilton are verify_robot: true now.